### PR TITLE
Update minimal_access.sls

### PR DESCRIPTION
**Pull request by puneetk**
Remove for loop in favor of names

### CODE REVIEW

**Comment by nmadhok** [comment]
I'm not sure if this is the right way to go. Thoughts?

**Comment by puneetk** [comment]
does it not work or a stylistic issue. 

## 

Puneet

File: os-hardening/minimal_access.sls [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {% from "os-hardening/map.jinja" import hardening with context %}
-{% for dir in hardening.read_only_folders %}
-{{dir}}:
+hardening-read-only-folders:
   file.directory:
+    - names: {{ hardening.read_only_folders }}
```
Comment (nmadhok) [inline os-hardening/minimal_access.sls:4]: `names` does not exist as an argument to `file.directory`: http://docs.saltstack.com/en/latest/ref/states/all/salt.states.file.html#salt.states.file.directory

Comment (puneetk) [inline os-hardening/minimal_access.sls:4]: hmm my bad, my assumption was names is global to all state functions 

## 

Puneet

**Comment by puneetk** [comment]
@nmadhok 
It works fine in 2014.7+ versions 

```
# -*- coding: utf-8 -*-
# vim: ft=sls
{% set read_only_folders = ['/usr/local/sbin','/usr/local/bin','/usr/sbin','/usr/bin','/sbin','/bin'] %}
hardening-read-only-folders:
  file.directory:
    - names: {{ read_only_folders }}
    - user: root
    - group: root
    - follow_symlinks: True
    - dir_mode: 755
    - file_mode: 755
    - recurse:
      - user
      - group
      - mode
```

which works like so, so looks like names is valid construct to use in file, let me know. 

```
root@salt:/srv/salt# salt 'onion' state.sls test
onion:
  Name: /bin - Function: file.directory - Result: Changed
  Name: /sbin - Function: file.directory - Result: Changed
  Name: /usr/bin - Function: file.directory - Result: Changed
  Name: /usr/sbin - Function: file.directory - Result: Changed

Summary
------------
Succeeded: 4 (changed=4)
Failed:    0
------------
```

**Comment by nmadhok** [comment]
My bad. I didn't think about the recent versions. I was thinking more about supporting users using older versions. Never mind. Thanks for the change!

**Comment by puneetk** [comment]
should we add a disclaimer for supported versions ? I can write one up 

## 

Puneet

**Comment by nmadhok** [comment]
That would be great!

**Pull request merged**